### DISCUSSION TITLE
Formatting/styling input field (material)

### DISCUSSION THREAD

**Discussion by zoltanhalasz** [ngx-formly/ngx-formly#2812] [Q&A] [answered]
I have a simple login using formly in angular. (using material styling)
How can I add a border to each input, to appear in a box?
I want email and password to be shown in a rounded box, but I am not able to target them.
![image](https://user-images.githubusercontent.com/42749923/115869936-4019d480-a447-11eb-8222-ecd788820915.png)

**Comment by aitboudad** [comment, marked as answer]
You can use `outline` appearance:

```ts
{
  type: 'email',
  templateOptions: {
    appearance: 'outline', // 'legacy' | 'standard' | 'fill' | 'outline'
}
```
You could also define it globally https://github.com/ngx-formly/ngx-formly/issues/1144#issuecomment-493927991

For a CSS solution, just target the component type selector, you could also use `className` for a specific field:
```ts
{
  className: 'custom_class',
}
```
